web/federate.go

```diff
@@ -62,6 +70,14 @@ func (fed *Federation) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		Type:   dto.MetricType_UNTYPED.Enum(),
 	}
 
+	numGlobLabels := len(fed.globalLabels)
+	for ln, lv := range fed.globalLabels {
```

Comment:
This is not the same behaviour as over in alerting, as I think this can lead to having a label twice whereas we want to to be a default if none is provided.